spring-cloud-services-spring-connector/src/main/java/io/pivotal/spring/cloud/config/java/ServiceInfoPropertySourceAdapter.java

```diff
@@ -88,8 +88,13 @@ private void conditionallyExcludeRabbitAutoConfiguration() {
 	private boolean appIsBoundToRabbitMQ() {
 		boolean hasEnvironmentBinding = environment.containsProperty("spring.rabbitmq.host") &&
 				!environment.getProperty("spring.rabbitmq.host").equals("localhost");
-		boolean hasCloudBinding = !cloud.getServiceInfos(ConnectionFactory.class).isEmpty();
-
+		
+		boolean hasCloudBinding = false;
+		try {
+			Class<?> rabbitCFClass = Class.forName("org.springframework.amqp.rabbit.connection.ConnectionFactory");
```

Comment:
I think we also need to remove the `import` of this class on line 6 to fully get rid of this problem

Reply:
Ah...good point. Removing